pkg/apply/apply.go

```diff
@@ -670,12 +670,12 @@ func (t *TopicApplier) updatePlacement(
 			return err
 		}
 		if !result {
-			log.Info(
-				"Desired strategy is in-rack, but leaders aren't balanced. It is strongly suggested to do the latter first.",
+			log.Infof(
+				"Desired strategy is #{desiredPlacement}, but leaders aren't balanced. It is strongly suggested to do the latter first.",
```

Comment:
Woah, does this string templating work?

Reply:
heh.. I suspect not :)   Dangers of using GoLand -- I saw that formatting in the file, but when I look more closely it shows lines being compressed and thus the content being transmogrified.  Not sure I like that!!

Reply:
Example:

![image](https://user-images.githubusercontent.com/1111441/140573662-cd97cf08-13f1-45d4-88eb-1441a15b4594.png)